app/views/spree/reviews/create.js.erb

```diff
@@ -0,0 +1,13 @@
+<% if @review.errors.blank? %>
+	$("#notice-msg").html('<%= Spree.t('pages.product_detail.review.submit_successfuly')%>');
+	$("#notice-msg").addClass("alert alert-success mb-0");
+	$("#notice-msg").fadeOut(10000);
+	setTimeout(function(){
+	    $('#notice-msg').fadeOut(1000, function(){
+	      location.reload(true);
```

Comment:
Remove the Location Reload .. We don't need to reload the location.

Reply:
it added because when user add more then one reviews the flash message is displayed every time using location.reload.